frameworks/Swift/swift-nio/benchmark_config.json

```diff
@@ -0,0 +1,24 @@
+{
+  "framework": "swift-nio",
+  "tests": [{
+    "default": {
+      "json_url": "/json",
+      "plaintext_url": "/plaintext",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Platform",
+      "database": "None",
+      "framework": "SwiftNIO",
```

Comment:
The code doesn't look like it is using a framework, to me (see examples like Rails where a framework library sits between the language built-ins and the application code).

I think this should be `"None"`

Reply:
You are right @msmith-techempower ! Thank you. I used the same config as the netty benchmarks, so maybe those need to get fixed as well. I have updated the PR.